src/config/importVafConfig.nim

```diff
@@ -0,0 +1,24 @@
+import json
+import os
+import strutils
+
+import initVafConfig
+import parseVafConfig
+
+import VafConfigStructure
+
+var homeDir: string = os.getHomeDir()
+
+proc importVafConfig*(): VafConfigStructure = 
+    var configExists: bool = fileExists(homedir & "/.vaf.json")
```

Comment:
this is also quite repetitive

Reply:
What is repetitive? The filepath?

Reply:
> What is repetitive? The filepath?

that and "configExists" is in almost every config file